src/components/graph/nodes/network-modification-node.js

```diff
@@ -40,6 +40,9 @@ const useStyles = makeStyles((theme) => ({
     buildStatusInvalid: {
         color: 'indianred',
     },
+    margin: {
+        marginLeft: '10px',
```

Comment:
Use theme.spacing(X) instead of pixels ?
theme.spacing(1.25) = 10px;

Reply:
Done